src/test/java/codeu/controller/ActivityFeedServletTest.java

```diff
@@ -0,0 +1,78 @@
+package codeu.controller;
+
+import codeu.model.store.basic.UserStore;
+import codeu.model.data.User;
+
+import java.io.IOException;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import javax.servlet.http.HttpServletResponse;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class ActivityFeedServletTest {
+
+    private ActivityFeedServlet activityFeedServlet;
+    private HttpServletRequest mockRequest;
+    private RequestDispatcher mockRequestDispatcher;
+    private HttpSession mockSession;    
+    private HttpServletResponse mockResponse;
+    private UserStore mockUserStore;
+    private User mockUser;
+
+    @Before
+    public void before() {
+        activityFeedServlet = new ActivityFeedServlet();
+
+        mockRequest = Mockito.mock(HttpServletRequest.class);
+        mockRequestDispatcher = Mockito.mock(RequestDispatcher.class);                
+        mockSession = Mockito.mock(HttpSession.class);
+        mockResponse = Mockito.mock(HttpServletResponse.class);
+        mockUserStore = Mockito.mock(UserStore.class);
+
+        Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/activityfeed.jsp"))
+        .thenReturn(mockRequestDispatcher);
+        Mockito.when(mockRequest.getSession()).thenReturn(mockSession);
+        activityFeedServlet.setUserStore(mockUserStore);
+    }
+
+    @ Test
```

Comment:
There shouldn't be a space between @ and Test. Also there shouldn't be an empty line between the @Test annotation and the method. (Ditto to all the tests below.)

Reply:
Fixed!